charts/victoria-metrics-anomaly/Chart.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v1
 name: victoria-metrics-anomaly
 description: Victoria Metrics Anomaly Detection - a service that continuously scans Victoria Metrics time series and detects unexpected changes within data patterns in real-time.
 version: 0.3.1
-appVersion: v1.1.0
+appVersion: v1.2.1
```

Comment:
@dima-vm , I may not have the context, but shouldn't we bump it to `1.5.x+` to explicitly enforce licensing?